styles.css

```diff
@@ -0,0 +1,502 @@
+    font-weight: bold;
+    /* Licht springen van knop */
+    transform: scale(1.05);
+}
+
+/* --------------- 6.2 Hello (Intro text and Contactlink text) ------------------- */
+
+.hello {
+    text-align: center;
+    display: flex;
+    flex-flow: column;
+    gap: 1.5rem;
+    padding: 2rem;
+}
+
+.info-banner {
+    text-align: center;
+    font-family: var(--f2);
+    font-weight: 700;
+    color: var(--text2-color);
+    margin-bottom: 1.5rem;
+}
+
+.info-banner h5 {
+    margin: 0 0 1rem; /* Voegt ruimte toe tussen paragrafen */
+    line-height: 1.5; /* Stelt de hoogte van de lijnen in */
+}
+
+.contact-link {
+    font-weight: 400;
+    font-size: 1em
+}
+
+.contact-link > a {
+    text-decoration: none;
+    color: var(--text2-color);
+}
+
+
+/*main-content-wrapper {*/
+/*    display: flex;*/
+/*    flex-flow: column;*/
+/*    max-width: clamp(320px, 80%, 1200px);*/
+/*    justify-content: center;*/
+
+
+/* --------------- 6.3 main-content ------------------- */
+
+/* 6.3.1 Main Articles (Project Showcase) */
+
+.projects-section {
+    display: flex;
+    flex-direction: column;
+    gap: 2rem; /* Ruimte tussen projecten */
+    padding: 2rem;
+    background-color: var(--alt-color);
+}
+
+.projects-section {
+    display: flex;
+    flex-direction: column;
+    gap: 2rem; /* Ruimte tussen projecten */
+    padding: 2rem;
+    background-color: var(--alt-color);
+}
+
+.project {
+    display: flex;
+    flex-direction: row-reverse; /* Standaard layout: afbeelding en tekst naast elkaar */
+    gap: 1rem;
+    align-items: center;
+    background-color: var(--antiflash-white);
+    padding: 1rem;
+    border: 1px solid var(--scol);
+    border-radius: 8px;
+    box-shadow: 0 4px 6px rgba(0, 0, 0, 0.1);
+}
+
+.order2 {
+    flex-direction: row; /* Keert de volgorde om: afbeelding rechts */
+}
+
+.project img {
+    width: 40%; /* Pas aan voor gewenste breedte */
+    height: auto;
+    border-radius: 8px;
+}
+
+.project-content {
+    display: flex;
+    flex-direction: column;
+    gap: 2rem;
+    flex: 1; /* Tekst vult resterende ruimte */
+    padding: 3em;
+}
+
+.project-content h3 {
+    color: var(--text-color);
+    font-size: 1.5rem;
+    margin-bottom: 0.5rem;
+
+}
+
+.project-content p {
+    padding-top: 2em;
+    font-size: 0.9em;
+    font-weight: 500;
+    color: var(--text-color);
+    line-height: 1.5;
+}
+
+.projectview {
+    color: var(--main-color);
+    text-transform: uppercase;
+    font-weight: 700;
+    font-size: 0.8em;
+}
+
+/* 6.3.2 Main Info (Company Services) */
+
+.services-wrapper {
+    display: flex;
+    flex-wrap: wrap; /* Zorgt ervoor dat items naar de volgende rij gaan */
+    justify-content: center; /* Center alle blokken horizontaal */
+    gap: 2rem; /* Ruimte tussen de blokken */
+    margin: 10%;
+}
+
+.serviceblock {
+    flex: 1 1 calc(100% / 3 - 2rem); /* 3 kolommen, ruimte aftrekken voor de gap */
+    display: flex;
+    flex-direction: column; /* Zorg dat tekst van boven begint */
```

Comment:
Misschien heb ik het mis, maar ik denk dat flex-box binnen de blokjes niet nodig is omdat tekst van nature al onder elkaar komt te staan en links begint!